core/templates/dev/head/pages/learner-dashboard-page/learner-dashboard-page.controller.ts

```diff
@@ -266,8 +266,8 @@ angular.module('oppia').directive('learnerDashboardPage', [
                 ctrl.startIncompleteCollectionIndex = Math.max(
                   ctrl.startIncompleteCollectionIndex - ctrl.PAGE_SIZE, 0);
               }
-            } else if (
-              section === LEARNER_DASHBOARD_SECTION_I18N_IDS.COMPLETED) {
+            } else if (section ===
+                LEARNER_DASHBOARD_SECTION_I18N_IDS.COMPLETED) {
```

Comment:
The line break should be after the '('. Why are you changing it?

If you need to, introduce a second line break after the '==='.

Reply:
Done